LoopKit Example/MasterViewController.swift

```diff
@@ -89,8 +89,8 @@ class MasterViewController: UITableViewController {
             switch ConfigurationRow(rawValue: indexPath.row)! {
             case .basalRate:
                 cell.textLabel?.text = LocalizedString("Basal Rates", comment: "The title text for the basal rate schedule")
-            case .glucoseTargetRange:
-                cell.textLabel?.text = LocalizedString("Glucose Target Range", comment: "The title text for the glucose target range schedule")
+            case .correctionRange:
+                cell.textLabel?.text = LocalizedString("Correction Range", comment: "The title text for the glucose correction range schedule")
```

Comment:
[nit] In the comment, "glucose correction range schedule" to "correction range schedule".

Reply:
I think glucose correction range is appropriate here.  Correction range is just a shortened version.